docs/articles/documentation/recipes/index.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: docs
+title: Recipes
+permalink: /documentation/recipes/
+---
+# Recipes
+
+This section provides examples and recipes on how to use TestCafe in different scenarios.
```

Comment:
> recipes on

Are you sure we can say `recipes on smth`? I don't know, just asking, but my feeling is that we can't.